documentacion/posintegrado/README.md

```diff
@@ -0,0 +1,275 @@
+# POS Integrado
+
+## Cómo empezar
+
+El SDK para POS Integrado, cuenta de 2 partes, la primera pieza es una librería/SDK escrito en C,
+el cual es multi plataforma, y debe ser instalado en tu maquina para poder ser utilizado por la
+segunda pieza, la cual mediante un Wrapper, accede a las funciones disponibles en la librería C.
```

Comment:
There are a lot of commas there (and a single long sentence!). Maybe you can let the reader take a break somewhere in the middle of that paragraph :)